packages/remix-cloudflare-workers/package.json

```diff
@@ -16,9 +18,9 @@
     "@remix-run/server-runtime": "1.0.6"
   },
   "peerDependencies": {
-    "@cloudflare/workers-types": "^2.2.2"
+    "@cloudflare/workers-types": "^3.2.0"
```

Comment:
This would be a breaking change for `@remix-run/cloudflare-workers`

Reply:
I didn't notice any breaking changes in the types, but I might have missed something. Seems like they are just additions.

Reply:
It doesn't matter if the API didn't change, people using `@remix-run/cloudflare-workers` would need to use a new major version of `@cloudflare/workers-types` so this will be a breaking change (as is every change to the `peerDependencies`)